firefox-ios/RustFxA/RustFirefoxAccounts.swift

```diff
@@ -122,7 +111,7 @@ open class RustFirefoxAccounts {
         return RustFirefoxAccounts.prefs?.boolForKey(PrefsKeys.KeyEnableChinaSyncService) ?? AppInfo.isChinaEdition
     }
 
-    private func createAccountManager(features: RustFxAFeatures) -> FxAccountManager {
+    private func createAccountManager(prefs: Prefs) -> FxAccountManager {
         let prefs = RustFirefoxAccounts.prefs
```

Comment:
not sure how this works in swift. Does the new method signature (feature => prefs) get hidden by this?